PD-176 Create Controller or Node Reference Article (#1842)
truenas/documentation: content/References/ControllerOrNode.md

---
title: "Controller or Node?"
description: "This article defines nodes and controllers, and explains how these apply to TrueNAS storage arrays and clusters."
weight: 
aliases:
tags:
- scaleclustering
---

## Controller or Node?

> In general, terms like controller and node are sometimes used to mean the same thing but are different when talking about TrueNAS storage arrays and configuration scenarios.

Reviewer: ```suggestion
The terms controller and node have different meanings when referring to TrueNAS storage arrays and their configuration scenarios.
```
